jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -118,11 +118,7 @@ def process(self, input, *args, **kwargs):
         fit_sip_hdr = {k.lower(): v for k, v in fit_sip_hdr.items()}
 
         # update meta.wcs_info with fit keywords except for naxis*
-        for key in ['naxis1', 'naxis2']:
-            del fit_sip_hdr[key]
-
-        # update meta.wcs_info with fit keywords
-        result.meta.wcsinfo.instance.update(fit_sip_hdr)
+        del fit_sip_hdr['naxis*']
```

Comment:
This doesn't work, raises an error.

Reply:
Yes, I just moved this before the line in which the header is converted to a dictionary.